app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
@lpichler Now that you're loading `Chargeback` in the tests can you go back to using `DescendantLoader.instance.load_subclasses(self)` and see if the tests goes green?